airbyte-integrations/connectors/source-rki-covid/source_rki_covid/source.py

```diff
@@ -395,6 +432,199 @@ def path(
         return "germany/history/hospitalization/"
 
 
+# STATES FULL-REFRESH | INCREMENTAL
+# source: states/history/cases/:days | FULL-REFRESH
+class StatesHistoryCases(RkiCovidStream):
+    """Docs: https://api.corona-zahlen.org/germany/states/history/cases/:days"""
+
+    primary_key = ""
+
+    def __init__(self, config, **kwargs):
+        super().__init__(**kwargs)
+        self.start_date = config.get("start_date")
+
+    def date_to_int(self, start_date) -> int:
+        diff = datetime.now() - datetime.strptime(start_date, "%Y-%m-%d")
+        if diff.days <= 0:
+            return 1
+        return diff.days
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        if response.json():
+            for key, value in response.json().get("data").items():
+                for x in value.get("history"):
+                    record = {"name": value.get("name"), "state": key, "cases": x.get("cases"), "date": x.get("date")}
```

Comment:
To follow up on https://github.com/airbytehq/airbyte/pull/17607#discussion_r1006064120, this here would look like:

```suggestion
                    x.update({"name": value.get("name"), "abbreviation": key})
```

or 
```suggestion
                    x.update({"state_name": value.get("name"), "state_abbreviation": key})
```